templates/brew-build.yml

```diff
@@ -61,7 +61,8 @@ $[[ inputs.stage ]]:build:brew:
         gsed -e /sha256/d -e "/url /s|\".*\"|\"https://github.com/mendersoftware/${REPO}.git\",|" -e "/url \"https/a\ \ \ \ revision: \"${CI_COMMIT_SHA}\",\n    using: GitHubPRDownloadStrategy\n  version \"1234\"" < ${FORMULA_NAME}_upstream.rb >> $FORMULA_FILE
     # 4. run brew build and test
     - HOMEBREW_NO_INSTALL_FROM_API=1 brew reinstall --build-from-source --verbose --debug --formula ./${FORMULA_FILE} < /dev/null
-    - brew test ./${FORMULA_FILE}
+    # FIXME: QA-1350
+    # - brew test mender-cli
```

Comment:
Instead of just commenting here, I have two other suggestions:

* Use `when: never` at the top so that the job is not added to the pipelines
* Put in the `before_script` a echo saying "this CI job is disabled due to QA-XXXX" and exit 1. Then add allow_falure: true. This way the CI job will show with the warning sign.

The main reason is visibility. As it stands, looking at the pipeline will seem that everything is okey green.